assets/terraform-modules/aws/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -173,3 +173,22 @@ variable "enable_csi" {
   type        = bool
   default     = false
 }
+
+
+variable "cpu_manager_policy" {
+  description = "CPU Manager policy to use for the worker pool. Possible values: `none`, `static`."
+  default     = "none"
+  type        = string
+}
+
+variable "kube_reserved_cpu" {
+  description = "CPU cores reserved for the Worker Kubernetes components like kubelet, etc."
+  default     = "300m"
+  type        = string
+}
+
+variable "system_reserved_cpu" {
+  description = "CPU cores reserved for the host services like Docker, sshd, kernel, etc."
+  default     = "1500m"
```

Comment:
This together seems a lot if you use an instance, which has only 2 cores for example. It leaves only 200m for controlplane.

Reply:
Is 2 core machine feasible for production use? Also this change is only for workers not controller nodes.

Reply:
With 1.5 core overhead, probably not :upside_down_face: I'm not sure, but I feel we can do better at estimating it + make it configurable.